operations/payments.md

```diff
@@ -190,3 +190,272 @@ Adds a new alternative payment to a specified customer.
+                "GrossValue": -3700.0,
+                "TaxValues": [],
+                "Breakdown": {
+                    "Items": [
+                        {
+                            "TaxRateCode": null,
+                            "NetValue": -3700.0,
+                            "TaxValue": 0.0
+                        }
+                    ]
+                }
+            },
+            "Notes": null,
+            "SettlementId": null,
+            "ClosedUtc": null,
+            "ConsumedUtc": "2023-03-02T12:12:35Z",
+            "ChargedUtc": "2023-03-06T07:31:52Z",
+            "CreatedUtc": "2023-03-06T07:31:51Z",
+            "UpdatedUtc": "2023-03-06T07:31:53Z",
+            "AccountingState": "Open",
+            "State": "Charged",
+            "Identifier": "ch_764309db-4bcd-4f2c-ad6a-1c178089deec",
+            "PaymentType": "CreditCard",
+            "DataDiscriminator": "CardData",
+            "Data": {
+                "CardData": {
+                    "PaymentCardId": "c922266b-291d-4e25-9df7-afbd007c1991"
+                },
+                "InvoiceData": null
+            }
+        }.
+        {
+            "Id": "be922eb7-bc5f-4877-b847-1120c0c2acd2",
+            "AccountId": "4ce18db7-3444-460a-b8af-afb900c92864",
+            "BillId": "d23ac52f-9b86-4a03-a6fe-5822dfcfc5c4",
+            "AccountingCategoryId": null,
+            "Amount": {
+                "Currency": "EUR",
+                "NetValue": -300.00,
+                "GrossValue": -300.00,
+                "TaxValues": [],
+                "Breakdown": {
+                    "Items": [
+                        {
+                            "TaxRateCode": null,
+                            "NetValue": -300.00,
+                            "TaxValue": 0.0
+                        }
+                    ]
+                }
+            },
+            "OriginalAmount": {
+                "Currency": "EUR",
+                "NetValue": -300.00,
+                "GrossValue": -300.00,
+                "TaxValues": [],
+                "Breakdown": {
+                    "Items": [
+                        {
+                            "TaxRateCode": null,
+                            "NetValue": -300.00,
+                            "TaxValue": 0.0
+                        }
+                    ]
+                }
+            },
+            "Notes": null,
+            "SettlementId": null,
+            "ConsumedUtc": "2023-03-02T12:12:35Z",
+            "ClosedUtc": "2023-03-02T12:12:35Z",
+            "ChargedUtc": "2023-03-02T12:12:32Z",
+            "CreatedUtc": "2023-03-02T12:12:32Z",
+            "UpdatedUtc": "2023-03-02T12:12:37Z",
+            "AccountingState": "Closed",
+            "State": "Charged",
+            "Identifier": "",
+            "PaymentType": "Cash",
+            "DataDiscriminator": null,
+            "Data": null
+        }
+    ],
+    "Cursor": "d98c9611-0006-4691-a835-af2e00b170c4"
+}
+```
+
```

Comment:
There is no definition for the response. I know it's pretty trivial, but for consistency there should be a table defining the response, consisting of the "Payments" array of Payment objects, plus "Cursor" for pagination.